### PR TITLE
BIG: Sword of Wealth and Power & Nexus of Becoming

### PR DESCRIPTION
**Pull request by dracontes**
Tested card scripts for:
- [Sword of Wealth and Power](https://scryfall.com/card/big/26/sword-of-wealth-and-power)
- [Nexus of Becoming](https://scryfall.com/card/big/25/nexus-of-becoming)

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/sword_of_wealth_and_power.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name:Sword of Wealth and Power
+ManaCost:3
+Types:Artifact Equipment
+K:Equip:2
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ 2 | AddToughness$ 2 | AddSVar$ SwordOfWealthAndPowerCE | AddKeyword$ Protection:Instant:instants & Protection:Sorcery:sorceries | Description$ Equipped creature gets +2/+2 and has protection from instants and from sorceries.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sword_of_wealth_and_power.txt:5]: write it as single protection KW

Comment (dracontes) [reply forge-gui/res/cardsfolder/upcoming/sword_of_wealth_and_power.txt]: Done. I take it this is another cleanup item on the to-do list.

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/sword_of_wealth_and_power.txt]: @Hanmac 
we prefer to split it for the engine or have the text closer to the print?

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/sword_of_wealth_and_power.txt]: Have two Separate keywords would be easier (there is also a Rule about that)

Also haven't I updated the Plural style?
So shouldn't `Protection:Instant` be enough?

Comment (dracontes) [reply forge-gui/res/cardsfolder/upcoming/sword_of_wealth_and_power.txt]: FWIW, this is what I get with `AddKeyword$ Protection:Instant & Protection:Sorcery:sorceries`, assuming the plural style doesn't cover 'sorcery'.
image

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/sword_of_wealth_and_power.txt]: I think it should, try leaving that part away

Comment (dracontes) [reply forge-gui/res/cardsfolder/upcoming/sword_of_wealth_and_power.txt]: Turns out the plural style does cover 'sorcery'. Edited the relevant file accordingly.

**Review by tool4ever — APPROVED** [review]

**Pull request merged**